fv3core/utils/grid.py

```diff
@@ -403,21 +361,23 @@ def overwrite_edges(self, var, edgevar, left_i_index, left_j_index):
     def compute_origin(self, add: Tuple[int, int, int] = (0, 0, 0)):
         return (self.is_ + add[0], self.js + add[1], add[2])
 
-    def default_origin(self):
+    def full_origin(self):
```

Comment:
should we reflect the difference between compute full origin in a short docstring here?

Reply:
I added docstrings, let me know what you think.

Reply:
looks good